### PR TITLE
Retrieve teams URL from vault

### PR DESCRIPTION
**Pull request by r-ash**

### CODE REVIEW

File: config/orderly-web.yml [hunk 1]
```diff
@@ -57,6 +57,7 @@ orderly:
     tag: master
   env:
     SLACK_URL: VAULT:secret/slack/orderly-webhook:url
+    TEAMS_URL: VAULT:secret/teams/orderly-webhook:url
```
Comment (richfitz) [inline config/orderly-web.yml:60]: Can you please add this to the VIMC vault?

**Review by richfitz — CHANGES_REQUESTED** [review]

File: config/orderly-web.yml (hunk 1 above)
Comment (r-ash) [reply config/orderly-web.yml:60]: Should already be in there - unless I have added it incorrectly?

Comment (r-ash) [reply config/orderly-web.yml:60]:
```
vault read secret/teams/orderly-webhook
Key                 Value
---                 -----
refresh_interval    768h
url                 ....
```

Comment (richfitz) [reply config/orderly-web.yml:60]: Sorry, you're right - not sure why that failed for me before 🤔

**Review by richfitz — APPROVED** [review]

**Pull request merged**